src/main/java/org/commcare/core/parse/CommCareTransactionParserFactory.java

```diff
@@ -80,20 +85,31 @@ public TransactionParser getParser(KXmlParser parser) {
             }
             req();
             return userParser.getParser(parser);
+        } else if (FixtureIndexSchemaParser.INDICE_SCHEMA.equalsIgnoreCase(name)) {
+            return new FixtureIndexSchemaParser(parser, fixtureSchemas);
         } else if ("fixture".equalsIgnoreCase(name)) {
+            String id = parser.getAttributeValue(null, "id");
+            String isIndexedAttr = parser.getAttributeValue(null, "indexed");
+            boolean isIndexed = "true".equals(isIndexedAttr);
             req();
-            return fixtureParser.getParser(parser);
+            if (isIndexed) {
+                FixtureIndexSchema schema = fixtureSchemas.get(id);
```

Comment:
I purposefully designed it this way because the setup will still work it just won't have any meta data columns. I don't know if there is any use to this, but it would mean that instead of storing one huge fixture blob in one row of the normal fixture db it would create a dedicated table and store a bunch of little blobs for each entry in the fixture.